fdbmonitor/SimpleIni.h

```diff
@@ -324,15 +324,15 @@ class CSimpleIniTempl
 #endif
 
         /** Strict less ordering by name of key only */
-        struct KeyOrder : std::binary_function<Entry, Entry, bool> {
+        struct KeyOrder {
```

Comment:
This file is one of those third-party-don't-edit-it files. Please revert these changes (I'm assuming whether or not it's a good idea to include files we shouldn't edit in source control is out of scope for this PR)

Reply:
If I revert this it will break the Windows build. std::binary_function was removed in C++17. So either we leave the change in or upgrade this external dependency. Fo now I will leave this in - updating `SimpleIni.h` should be done in a different PR.